test/helpers.tcl

```diff
@@ -77,3 +77,7 @@ suppress_message PPL 60
 # suppress tap info messages
 suppress_message TAP 100
 suppress_message TAP 101
+
+# suppress cts info messages
+suppress_message CTS 36
+suppress_message CTS 37
```

Comment:
Why are these hidden?  They are generated in c++ and not language dependent.

Reply:
That  was my thought too, but it turns out that they seem to be suppressed in some (but not all) tcl log files and also in some (but not all) Python log files. If we remove these directives, then 9 of the regressions will differ, where one will have the messages and the other will not. It is not consistent, so I just disabled for both log files.

Reply:
That because you have a bug:

parms.setRunPostCtsOpt(post_cts_disable)

Reply:
The default value of `post_cts_disable` is `False`, so this should be exactly equivalent with `  cts::set_disable_post_cts [info exists flags(-post_cts_disable)]` in the tcl version of `clock_tree_synthesis` so will look at it a little more later.

Reply:
You are setting whether to run it to the the value of whether to disable it.